acceptance/lib/pxp-agent/bolt_pxp_module_helper.rb

```diff
@@ -81,65 +85,79 @@ def run_task(broker, targets, task, files, input: {}, metadata: nil, **kwargs, &
   do_module_action(broker, agent_identities(targets), 'task', 'run', params, **kwargs, &block)
 end
 
+def script_request(broker, agent, script, arguments, **kwargs, &block)
+  params = { arguments: arguments, script: script }
+  target = ["pcp://#{agent}/agent"]
+  do_module_action(broker, target, 'script', 'run', params, **kwargs, &block)
+end
+
 # Makes a non-blocking request to start a command.
 # Block is executed on the "response_dataset" object from do_module_action.
 def run_command(broker, targets, command, **kwargs, &block)
   params = { command: command }
   do_module_action(broker, agent_identities(targets), 'command', 'run', params, **kwargs, &block)
 end
 
+# Starts a download file execution.
+# Block is executed on the "response_dataset" object from do_module_action.
+def run_download_file(broker, agent, files, **kwargs, &block)
```

Comment:
The `run_command` method name matches the action name for running a command - it seems like this one should be named the same as the `download_file` action, too

Reply:
Yeah, the name for the download_file module doesn't match the other modules. If it were to match the other modules the module name itself should be `file` with an action for `download`. I was concerned about that name because on first glance it might appear like it'll perform more than it does (i.e. the file module does _not_ provide the user the ability to change file perms or file contents).